generators/dotnet-bots/templates/dotnet/bots/request-reply/RequestResponse.csproj

```diff
@@ -2,11 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
@symphony-thibault I put the latest version netcoreapp3.1 but I realize this may not be suitable for some customers. What should I put? Should I keep it the same event if it is an old version?